spec/requests/api_docs_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "API documentation", type: :request do
+  it "shows the OFN API v1" do
+    get rswag_ui_path
+    expect(response).to redirect_to "/api-docs/index.html"
+
+    get "/api-docs/index.html"
+    expect(response).to have_http_status :success
+
+    expect(response.body).to match "API V1"
+  end
+
+  it "can load the Swagger config" do
+    get "/api-docs/v1/swagger.yaml"
+    expect(response).to have_http_status :success
+  end
+end
```

Comment:
This is an integration test with an external component. So it's not meant to be complete or particularly sophisticated. I just reproduced the error we had. It would have been simpler in a system spec but they are slow. I didn't want to add the overhead if we can run this in "0.1276 seconds" and it's still very simple.